src/main/java/de/unistuttgart/iste/gits/content_service/controller/SectionController.java

```diff
@@ -42,8 +42,8 @@ public Section updateStageOrder(@Argument StageOrderInput input){
     }
 
     @QueryMapping
-    public List<Section> findSectionsByChapter(@Argument UUID id){
-        return sectionService.getSectionByChapterId(id);
+    public List<List<Section>> sectionsByChapterIds(@Argument List<UUID> chapterIds){
```

Comment:
Why replace this query?
Doesn't it make perfect sense to have a query for a single chapter?
Realistically, a user would only be viewing 1 chapter at a time, or am I mistaken?

Reply:
https://gits-enpro.readthedocs.io/en/latest/dev-manuals/architecture/graphql.html#graphql-mesh-concept

Please read this, especially the last bullet point